### PR TITLE
#161581971 Add guesthouse modal mobile responsiveness

### PR DESCRIPTION
**Pull request by janetnim**
#### What does this PR do?
Makes the guest house modal mobile responsive

#### Description of Task to be completed?
Create styling for media screens with minimum width of:
* 320px
* 480px
* 600px
* 768px
* 1024px

#### How should this be manually tested?
* Clone the repo `https://github.com/andela/travel_tool_front` and `https://github.com/andela/travel_tool_back`
* Navigate to the branch `ft-add-guesthouse-mobile-reponsiveness-161581971` in travel tool front and `develop` in travel_tool_back
* Run the following commands on both projects:
>`yarn install`
>`make start`
* On your browser test it with different mobile screens as in the gif in the screenshots section below.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[161581971](https://www.pivotaltracker.com/story/show/161581971)

#### Screenshots (if appropriate)

mobile

#### Questions:
N/A

### CODE REVIEW

**Comment by janetnim** [comment]
@matthewacha 
screen shot 2018-11-12 at 13 50 16

File: src/components/Forms/NewAccommodationForm/NewAccommodation.scss [hunk 1]
```diff
@@ -1,156 +1,148 @@
...
+form fieldset:last-child .submit-area {
+    position: relative;
+    bottom: 0;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    height: 71px;
+    width: 96.4%;
+    }
+@import 'NewAccommodation-smallMobileScreen';
```
Comment (amarachukwu-agbo) [inline src/components/Forms/NewAccommodationForm/NewAccommodation.scss]: Are partial imports not supposed to come at the beginning of the main file? Is there a reason for importing at the end?

**Comment by amarachukwu-agbo** [comment]
Good job @janetnim , responsiveness works across all specified screens. Please see my comment

File: src/components/Forms/NewAccommodationForm/NewAccommodation.scss (hunk 1 above)
Comment (janetnim) [reply src/components/Forms/NewAccommodationForm/NewAccommodation.scss]: Importing it at the end of the file did not work for me. The responsive stylings did not apply and the default styling was the one being used.

Comment (amarachukwu-agbo) [reply src/components/Forms/NewAccommodationForm/NewAccommodation.scss]: Okay

**Comment by amarachukwu-agbo** [comment]
@samuelkubai , LGTM

**Pull request merged**